build_daemon/lib/src/managers/build_target_manager.dart

```diff
@@ -30,15 +30,18 @@ class BuildTargetManager {
 
   Set<BuildTarget> get targets => _buildTargets.keys.toSet();
 
+  /// Returns all tracked channels of the manager.
```

Comment:
[nit] document like a field. Also avoid repeating information which is already clear from context.

/// All the tracked channels.